### ISSUE TITLE
no action sequences included in the input

### ISSUE THREAD

**Issue by schwgHao** [kyegomez/RT-X#10]
Thank you for the rtx1 implementation!
If I understand it correctly, the input does not include the corresponding action information for each frame. Therefore, when predicting future actions, one cannot refer to the historical action sequence. 
My question would be how much does this affect the results?

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/kyegomez) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/kyegomez/RT-X/issues/10">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/kyegomez/RT-X/issues/10/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/kyegomez/RT-X/issues/10/pledge.svg">
</picture>
</a>

**Comment by kyegomez** [comment]
@schwgHao well, the output now is logits of the interleaved images with text. It outputs text logits to be tokenized like llama.



Basically, to add the multi dimensional output as in the paper you would need to integrate token learner. There is an implementation in my zeta library if you would like to work make the model output 7 dimensional tensors.

https://zeta.apac.ai/en/latest/zeta/nn/modules/token_learner/
